recipes/fast-cdr/all/conanfile.py

```diff
@@ -0,0 +1,137 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import textwrap	
+
+class FastCDRConan(ConanFile):
+
+    name = "fast-cdr"
+    license = "Apache-2.0"
+    homepage = "https://github.com/eProsima/Fast-CDR"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "eProsima FastCDR library for serialization"
+    topics = ("conan", "DDS", "Middleware", "Serialization")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared":          [True, False],
+        "fPIC":            [True, False]
+    }
+    default_options = {
+        "shared":            False,
+        "fPIC":              True
+    }
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt"]
+    _cmake = None
+
+    @property
+    def _pkg_cmake(self):
+        return os.path.join(
+            self.package_folder,
+            "lib",
+            "cmake"
+        )
+
+    @property
+    def _module_subfolder(self):
+        return os.path.join(
+            "lib",
+            "cmake"
+        )
+
+    @property
+    def _module_file_rel_path(self):
+        return os.path.join(
+            self._module_subfolder,
+            "conan-target-properties.cmake"
+        )
+
+    @property
+    def _pkg_share(self):
+        return os.path.join(
+            self.package_folder,
+            "share"
+        )
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _create_cmake_module_alias_target(self):
+        content = ""
+        if self.options.shared and self.settings.os == "Windows":
+            # alias plus special properties which are required
+            content += textwrap.dedent("""\
+                if(TARGET fastcdr::fastcdr AND NOT TARGET fastcdr)
+                    add_library(fastcdr INTERFACE IMPORTED)
+                    set_property(TARGET fastcdr PROPERTY INTERFACE_LINK_LIBRARIES fastcdr::fastcdr)
+                    set_property(TARGET fastcdr PROPERTY INTERFACE_COMPILE_DEFINITIONS FASTCDR_DYN_LINK)
+                endif()
+                set_property(TARGET fastcdr::fastcdr PROPERTY INTERFACE_COMPILE_DEFINITIONS FASTCDR_DYN_LINK)
+            """)
+        else:
+            content += textwrap.dedent("""\
+                if(TARGET fastcdr::fastcdr AND NOT TARGET fastcdr)
+                    add_library(fastcdr INTERFACE IMPORTED)
+                    set_property(TARGET fastcdr PROPERTY INTERFACE_LINK_LIBRARIES fastcdr::fastcdr)
+                endif()
+            """)
+        tools.save(os.path.join(self.package_folder, self._module_file_rel_path), content)
+
+    def _get_configured_cmake(self):
```

Comment:
Could you please stick to out usual name? My eyes get stuck on this line every time


```suggestion
    def _configure_cmake(self):
```